src/index.js

```diff
@@ -22,9 +23,9 @@ class RequestsRouting extends React.Component {
   NoMatch() {
     return (
       <div>
-        <h2>Uh-oh!</h2>
+        <h2><FormattedMessage id="ui-requests.routingErrorOops" /></h2>
         <p>
-          How did you get to
+          <FormattedMessage id="ui-requests.routingError" />
           {' '}
           <tt>{this.props.location.pathname}</tt>
           ?
```

Comment:
Please, add this to routingError as value
` {' '}<tt>{this.props.location.pathname}</tt> ? `